table.go

```diff
@@ -50,16 +55,23 @@ type Table struct {
 }
 
 // DelTable deletes a specific table, along with all chains/rules it contains.
-func (cc *Conn) DelTable(t *Table) {
+func (cc *Conn) DelTable(t *Table, force ...bool) {
 	cc.mu.Lock()
 	defer cc.mu.Unlock()
 	data := cc.marshalAttr([]netlink.Attribute{
 		{Type: unix.NFTA_TABLE_NAME, Data: []byte(t.Name + "\x00")},
 		{Type: unix.NFTA_TABLE_FLAGS, Data: []byte{0, 0, 0, 0}},
 	})
+
+	var hdrType netlink.HeaderType
```

Comment:
nit: I would prefer expressing this code as follows:

```go
	hdrType := delTableHeaderType
	if len(force) > 0 && force[0] {
		hdrType = destroyTableHeaderType
	}
```

This way, we initialize the variable to its default (a little clearer), and we only consider the first bool passed to the function.

(But maybe this comment is moot after addressing my other comment.)